earth_enterprise/rpms/build.gradle

```diff
@@ -251,20 +254,54 @@ task openGeeCommonDeb (type: Deb) {
 task openGeeServerRpm (type: Rpm) {
     requires('opengee-common', openGeeVersion, GREATER | EQUAL)
     packageName = 'opengee-server'
+    release = '1'
+    version = openGeeVersion
+    user = 'root'
+    permissionGroup = 'root'
+    packageGroup = 'Application/Productivity'
+
+    directory('/opt/google/install', 0700)
+    directory('/opt/google/gehttpd/logs', 0755)
+    directory('/opt/google/share/geplaces', 0755)
```

Comment:
Any reason geplaces is included here but searchexample is not?

Reply:
I did this actually before I added the _geplaces install into, so this probably should have been removed.